operators/pkg/instctrl/controller_test.go

```diff
@@ -89,7 +89,8 @@ var _ = Describe("The instance-controller Reconcile method", func() {
 		tenant := clv1alpha2.Tenant{
 			ObjectMeta: metav1.ObjectMeta{Name: testName},
 			Spec: clv1alpha2.TenantSpec{
-				Email: "test@email.me",
+				Email:     "test@email.me",
+				LastLogin: metav1.Now(),
```

Comment:
This change is not required, and shall be reverted.

Reply:
Reverted version does not pass tests: changed to `metav1.Time{Time: time.Now().AddDate(-3, 0, 0)}` (i.e. login three months ago). I would assume this represents something more similar to the imagined test case. Otherwise I would ask for a suggestion to which to revert that will pass the tests